pkg/kube/configmap/configmap_builder.go

```diff
@@ -32,12 +33,22 @@ func (b *builder) SetOwnerReferences(ownerReferences []metav1.OwnerReference) *b
 	return b
 }
 
+func (b *builder) SetLabels(labels map[string]string) *builder {
+	newLabels := make(map[string]string)
+	for k, v := range labels {
+		newLabels[k] = v
+	}
+	b.labels = newLabels
+	return b
+}
+
```

Comment:
If we use SetLabels multiple times in a builder object. Should we keep only the labels from the last function call or merge all the maps and labels together?

Reply:
That's a good question, I think with a name like `Set` we indicate that this will overwrite all labels, if we need the functionality to add labels incrementally this might be an `AddLabel` function.